### PR TITLE
remove explorer

### LINKED ISSUE #240: Remove `explorer` from D3N repo
We are going with our `scan`! 
- Remove the folder
- Update README
- Update docker configs

### PR DESCRIPTION
**Pull request by evilpeach**
Fixes #240

### CODE REVIEW

**Comment by sorawit** [comment]
Did you do all three the following points, as specified on the issue? 
- Remove the folder
- Update README
- Update docker configs

File: README.md [hunk 1]
```diff
@@ -8,15 +8,14 @@ This repository is a [monorepo] containing the reference implementation of D3N a
...
-| [`owasm`](owasm)       | 🔮 WebAssembly library for writing oracle scripts         |
-| [`scan`](scan)         |                                                           | 
-| [`spec`](spec)         | 📖 D3N research and specification knowledge base          |
-| [`studio`](studio)     | 🎬 In-browser IDE for testing and deploying owasm scripts |
+| Module               | Description                                               |
+| -------------------- | --------------------------------------------------------- |
+| [`chain`](chain)     | 🔗 D3N blockchain reference implementation                |
+| [`bridges`](bridges) | 📡 Lite client bridges on other smart contract platforms  |
+| [`owasm`](owasm)     | 🔮 WebAssembly library for writing oracle scripts         |
+| [`scan`](scan)       |                                                           |
```
Comment (sorawit) [inline README.md]: Can you add `🔍 Web interface to explore D3N network` to scan?

**Review by sorawit — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (evilpeach) [reply README.md]: Sure

**Review by sorawit — APPROVED** [review]
Looks good

**Review by prin-r — APPROVED** [review]

**Pull request merged**